### DISCUSSION TITLE
阿里云盘Open出现大流量下载

### DISCUSSION THREAD

**Discussion by fengymi** [AlistGo/alist#5451] [Q&A]
阿里云盘Open经常出现大流量下载，基本有几十G。倒没发现其实使用问题
![image](https://github.com/alist-org/alist/assets/17907299/91d41a39-f6b0-436d-aa87-d7cb26261809)
![image](https://github.com/alist-org/alist/assets/17907299/d1eefd95-a4d6-401a-ba28-94701d5250d5)
![image](https://github.com/alist-org/alist/assets/17907299/0834bbad-d565-469f-8a5c-9870d2266620)
![image](https://github.com/alist-org/alist/assets/17907299/f509de55-c0e0-4b8a-a57b-6fc6bff4f322)

![image](https://github.com/alist-org/alist/assets/17907299/152a5dee-33ee-4e71-8a7b-8dd440723dfb)

**Comment by anwen-anyi** [comment]
因为你开启了web代理，别人下载 你的本地就会下载

**Reply by fengymi** [reply]
别人下载为啥我本地会下载，我的alist没有别人使用
